### PR TITLE
implicit pattern

### PR DESCRIPTION
**Pull request by zhenglaizhang**

### CODE REVIEW

File: util/src/test/scala/util/MathTest.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+package util
+
+import util.Math.squareRoot
+
+import org.scalatest.FunSuite
+
+class MathTest extends FunSuite {
+
+  test("testSquareRoot") {
+    squareRoot.isDefinedAt(-1)
```
Comment (coding2world) [inline util/src/test/scala/util/MathTest.scala:11]: add assert ?

**Review by coding2world — CHANGES_REQUESTED** [review]

File: util/src/test/scala/util/MathTest.scala (hunk 1 above)
Comment (zhenglaizhang) [reply util/src/test/scala/util/MathTest.scala:11]: sure, will do

Comment (zhenglaizhang) [reply util/src/test/scala/util/MathTest.scala:11]: done

**Comment by zhenglaizhang** [comment]
@coding2world let me merge this again, so that I can kick off next commits

**Pull request merged**